src/js/gutenberg-plugins/commands.js

```diff
@@ -0,0 +1,71 @@
+import { useCommandLoader } from '@wordpress/commands';
+import { edit, settings } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { registerPlugin } from '@wordpress/plugins';
+
+const Commands = () => {
+	const getCommandLoader = () => {
+		const commands = [];
+		const excerptButton = document.querySelector(
+			'.editor-post-excerpt button'
```

Comment:
Nitpick: Maybe add some class to the button and add it here to make it more solid? This is just a minor note and not a blocker for merging this PR.

Reply:
Yeah, this is a good call out. I've added that